composer.json

```diff
@@ -5,26 +5,34 @@
   "license": "GPL-2.0",
   "require-dev": {
     "php": ">=7.0",
-    "wp-coding-standards/wpcs": "^1.0.0",
+    "wp-coding-standards/wpcs": "^1.1.0",
     "dealerdirect/phpcodesniffer-composer-installer": "^0.4.4",
     "wimg/php-compatibility": "^8.1",
     "phpunit/phpunit": "^6",
-    "brain/monkey": "^2"
+    "brain/monkey": "^2",
+    "codedungeon/phpunit-result-printer": "^0.20.3"
+  },
+  "require": {
+    "php": ">=7.0"
   },
   "config": {
     "platform": {
-      "php": "7.0"
-    }
+      "php": "7.1"
+    },
+    "process-timeout": 600,
+    "sort-packages": true
```

Comment:
The `sort-packages` will just sort the newly added packages in the `composer.json` by name, and the `process-timeout` is used if you are running the `phpunit` using composer. When you have a lot of tests, and if you are running the phpunit using the composer, it can timeout, so this will just expand this timeout so that the unit/integration test will pass.